site/app/templates/grading/electronic/DiscussionForumPanel.twig

```diff
@@ -9,11 +9,15 @@
 </div>
 
 <script>
-
 	
-		$('#posts_list').css('min-width', '-webkit-fill-available');
-		$(function() {
-			generateCodeMirrorBlocks(document);
-		});
+$('#posts_list').css('min-width', '-webkit-fill-available');
+$(function() {
+	generateCodeMirrorBlocks(document);
+});
+
+$(document).ready(function() {
+    loadAllInlineImages(true);
+
+});
```

Comment:
All of this code should probably be moved to the relevant `.js` file.  Perhaps `ta-grading.js`.